src/components/Hero/index.js

```diff
@@ -21,8 +21,8 @@ export const Hero = () => {
   const getData = useCallback(async () => {
     try {
       const trendingMovies = await getTrending();
-      setPosterURL(trendingMovies.results[0].poster_path);
-      setGenreIds(trendingMovies.results[0].genre_ids);
+      setPosterURL(trendingMovies.results[10].poster_path);
```

Comment:
What does this 10 mean? If we use this kind of arbitrary number we should have them as a constant with a descriptive name.